contract/ignition/modules/BankProxyModule.ts

```diff
@@ -13,13 +13,18 @@ const bankProxyModule = buildModule('BankProxyModule', (m) => {
   // We will upgrade this contract with a new version later.
   const bankContractImplementation = m.contract('Bank')
 
+  // Initialize function signature
+  const initialize = m.encodeFunctionCall(bankContractImplementation, 'initialize', [
+    '0x6c94e5dDC5a36F893b61257d9dd31BEf2117630f' // polygon deployed tips
+  ])
+
```

Comment:
i had to upgrade the `ethers` in contract folder because encodeFunctionCall is a new feature from hardhat ignition

Reply:
You talk about the dependency ?

Reply:
> You talk about the dependency ?

yes